RELEASE_NOTES.rst

```diff
@@ -17,32 +17,32 @@ coala 0.8.0 - grizzly
 
 ::
 
-                   -                         
-                 `Ns      :s-               
-            .     mMd`     :Nd.             
-           :h     /ss/`     +md.            
-           dN`    :NMMMy`  .ymmy. -+`       
-           dM+    dMMMMMm`.NMMMMN. +Mo      
-        `  -sddy: yMMMMMM/+MMMMMMo  dMo     
-       s/  +MMMMMy.dMMMMM-:MMMMMM+ -yhs`    
-      .Ms  /MMMMMMo /hdh:  oMMMMh`+MMMMm.   
-      -MN.  hMMMMMh  `/osssoos+-  dMMMMMs   
-       oyhho.+mMMm:+dMMMMMMMMMm+  sMMMMMs   
-       mMMMMMy``` dMMMMMMMMMMMMMh.`sMMMh`   
-       yMMMMMMy  `MMMMMMMMMMMMMMMMy:..`     
-       `yMMMMMd  yMMMMMMMMMMMMMMMMMMMMNh+`  
-         .ohhs-+mMMMMMMMMMMMMMMMMMMMMMMMMd  
-            .yMMMMMMMMMMMMMMMMMMMMMMMMMMMh  
-            mMMMMMMMMMMMMMMMMMMMMMMMMMMMh`  
-            yMMMMMMMMMMMMMMMNhssssyyyso-    
-             /dMMMMMMMMMNy+.                
-               ./syhys/-                    
+                   -
```

Comment:
So I tried resolving this, but for some reason, my editor automatically deletes and regenerates it. Should it be a problem considering there is no final effect on it ?